Client_Services/README.md

```diff
@@ -1,6 +1,39 @@
 # Client Services
-Every Asana Task has "Acceptance Criteria". Don't mark it as complete until they are met. If you don't know what those criteria are, ask.
-
 ## General Expectations
-All your expectations during this internship are best understood in the context of Software for Good's role as Client Services company.
-We serve our clients. Period. 
+All your expectations during this apprenticeship are best understood in the context of Software for Good's role as Client Services company. We serve our clients. Period.
+  
+**Clients**
+  
+Clients are the people (i.e. company) that we are developing the code for. Therefore, the client determines what tasks we do and approve of our work for a task. **A task isn't complete until the client says so.**
```

Comment:
> A task isn't complete until the client says so

i think there's a little more nuance to this that we probably should add 😛 working with clients is a bit of a dance, and there are times where we do need to push back, so i don't know that it's as clear cut as this statement kinda makes it seem. since we don't have a formal QA team, sometimes our team or the client's team will do testing and discover other things that need to be improved, or discover that they actually need a feature to do something else — i would consider these to be different tasks, even if the client says the original task isn't done until these new x, y, z things are done. 

clients do ultimately sign off on the work we've done, but there is more complexity that goes into managing tasks, especially in the Agile process we tend to run in

after chatting a bit with @SpraggleRock, here's what she suggested could go here instead:

> The clients are the people we are developing the app for. We work with clients to make sure the app we’re developing meets their needs and goals. How we work with them varies by client - sometimes they define and prioritize our tasks, and sometimes they are more hands off. 

so more of an explanation of our relationship with them and less focus on when things are "done" because that vary between clients and how involved they are 😄

Reply:
Thank you! This definitely sounds better.